packages/debugger/src/tokens.ts

```diff
@@ -616,36 +616,22 @@ export namespace IDebugger {
 
   /**
    * Select variable in the variables explorer.
+   *
+   * @hidden
+   *
+   * #### Notes
+   * This is experimental API
    */
   export interface IVariableSelection
     extends Pick<
       DebugProtocol.Variable,
       'name' | 'type' | 'variablesReference' | 'value'
     > {}
 
-  /**
-   * Debugger variables explorer interface.
-   */
-  export interface IVariablesPanel {
-    /**
-     * Select variable in the variables explorer.
-     */
-    latestSelection: IVariableSelection | null;
-    /**
-     * Variable view mode.
-     */
-    viewMode: 'tree' | 'table';
-  }
-
   /**
    * Debugger sidebar interface.
    */
-  export interface ISidebar extends Panel {
-    /**
-     * Debugger variables explorer.
-     */
-    variables: IVariablesPanel;
-  }
+  export interface ISidebar extends Panel {}
```

Comment:
Should `ISidebar` be removed?

```suggestion
```

Reply:
In 3.x, `ISidebar` is providing a public API. This is the reason I kept the `Panel` API. I assume the idea is for third-party extension to be able to add widgets in that panel.

https://github.com/jupyterlab/jupyterlab/blob/f4ef57707b65121888c3c87482eb3f3f9ab63425/packages/debugger/src/tokens.ts#L550-L565